schema/bom-1.7.xsd

```diff
@@ -7422,7 +7922,7 @@ limitations under the License.
                                             </xs:documentation>
                                         </xs:annotation>
                                     </xs:element>
-                                    <xs:element name="auth" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
incompatible change of types.
:x: this is a breaking change

Reply:
Same.